route.go

```diff
@@ -3,33 +3,130 @@ package netlink
 import (
 	"fmt"
 	"net"
+	"strings"
 	"syscall"
 )
 
-// Scope is an enum representing a route scope.
-type Scope uint8
-
-const (
-	SCOPE_UNIVERSE Scope = syscall.RT_SCOPE_UNIVERSE
-	SCOPE_SITE     Scope = syscall.RT_SCOPE_SITE
-	SCOPE_LINK     Scope = syscall.RT_SCOPE_LINK
-	SCOPE_HOST     Scope = syscall.RT_SCOPE_HOST
-	SCOPE_NOWHERE  Scope = syscall.RT_SCOPE_NOWHERE
-)
```

Comment:
I just don't want to cover the syscall pkg, becouse there are way more constans in syscall package that must be put in netlink pkg (for routes, rules, interfaces etc...), and i just thougth that netlink should be cover all syscall constans or none of them. But none of them is easier and i think more intuitive. For example values for FAMILY_V4, V6 are for me unintuitive, everyone who has programming some network stuff used to use AF_INET, AF_INET6 etc in all languages. Why introduce new naming convention for this values? In fact I want to remove them as well, but thought that maybe more peopel use it (more then Scope enum).